### PR TITLE
Update Ibexa Fastly doc

### PR DESCRIPTION
**Pull request by andrerom**
Link to Ibexa Fastly doc seems broken as title has changed, also added reminder about applying Ibexa's VCL files, where to find them and why they are needed.

### CODE REVIEW

File: docs/src/frameworks/ibexa/fastly.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ description: |
 
 ## Remove Varnish configuration
 
-Since eZ Platform 1.13.5, 2.4.3 and 2.5.0, Varnish is enabled by default when deploying on Platform.sh. In order to use Fastly, Varnish must be disabled:
```
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:13]: obsolete release, so just dropped it

File: docs/src/frameworks/ibexa/fastly.md [hunk 2]
```diff
@@ -49,6 +49,11 @@ platform variable:set -e production env:FASTLY_SERVICE_ID YOUR_ID_HERE
 platform variable:set -e production env:FASTLY_KEY YOUR_ID_HERE
 ```
 
+## Setup the correct VCL files
+
+There are two VCL files provided as starting point for use Fastly with Ibexa DXP, you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`.
```
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:54]:
```suggestion
There are two VCL files provided as starting point for use Fastly with Ibexa DXP, you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`. They handle varying cache by user context hash _(permissions)_ as well as several other needs by Ibexa DXP and it's underlying HttpCache system.
```

File: docs/src/frameworks/ibexa/fastly.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ description: |
 
 ## Remove Varnish configuration
 
-Since eZ Platform 1.13.5, 2.4.3 and 2.5.0, Varnish is enabled by default when deploying on Platform.sh. In order to use Fastly, Varnish must be disabled:
+Since eZ Platform 1.13.5 and 2.5.0, Varnish is enabled by default when deploying on Platform.sh. In order to use Fastly, Varnish must be disabled:
```
Comment (andrerom) [inline docs/src/frameworks/ibexa/fastly.md:13]:
```suggestion
In Ibexa DXP by default, Varnish is enabled by default when deploying on Platform.sh. In order to use Fastly, Varnish must be disabled:
```

**Comment by andrerom** [comment]
@Crell Who should review this?

File: docs/src/frameworks/ibexa/fastly.md [hunk 4]
```diff
@@ -49,6 +49,11 @@ platform variable:set -e production env:FASTLY_SERVICE_ID YOUR_ID_HERE
 platform variable:set -e production env:FASTLY_KEY YOUR_ID_HERE
 ```
 
+## Setup the correct VCL files
+
+There are two VCL files provided as starting point for use Fastly with Ibexa DXP, you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`. They handle varying cache by user context hash _(permissions)_ as well as several other needs by Ibexa DXP and it's underlying HttpCache system.
```
Comment (chadwcarlson) [inline docs/src/frameworks/ibexa/fastly.md:54]:
```suggestion
There are two VCL files provided as starting points for using Fastly with Ibexa DXP; you can find them in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/*.vcl`. They handle varying cache by user context hash _(permissions)_ as well as several other needs by Ibexa DXP and it's underlying HttpCache system.
```

**Comment by chadwcarlson** [comment]
LGTM @andrerom - one nit, but Ill merge once that passes.

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**